src/main/java/com/ericsson/ei/subscriptionhandler/RunSubscription.java

```diff
@@ -107,19 +107,25 @@ public boolean runSubscriptionOnObject(String aggregatedObject, Iterator<JsonNod
             while (conditionIterator.hasNext()) {
                 String rule = conditionIterator.next().get("jmespath").toString().replaceAll("^\"|\"$", "");
                 String new_Rule = rule.replace("'", "\"");
-                LOGGER.info("Rule : " + rule);
-                LOGGER.info("New Rule after replacing single quote : " + new_Rule);
+                LOGGER.debug("Rule : " + rule);
+                LOGGER.debug("New Rule after replacing single quote : " + new_Rule);
                 JsonNode result = jmespath.runRuleOnEvent(rule, aggregatedObject);
-                LOGGER.info("Result : " + result.toString());
-                if (result.toString() != null && result.toString() != "false" && !result.toString().equals("[]")) {
+                LOGGER.debug("Result : " + result.toString());
+                Boolean conclusion1 = !result.toString().equals("null");
+                Boolean conclusion2 = !result.toString().equals("false");
+                Boolean conclusion3 = !result.toString().equals("[]");
```

Comment:
I feel these Booleans can be renamed to conditions, just as in the logging output. More consistent and less confusing.